Essentials/Commands/CleanupModule.cs

```diff
@@ -280,8 +280,19 @@ public bool OwnedBy(MyCubeGrid grid, string str)
             {
                 var player = Utilities.GetPlayerByNameOrId(str);
                 if (player == null)
+                {
+                    if(long.TryParse(str, out long NPCId))
+                    {
+                        if(MySession.Static.Players.IdentityIsNpc(NPCId))
+                        {
```

Comment:
You'll get a null reference exception if this statement is false. Remove the else on L291